### PR TITLE
Support HTML tags inside system messages

### PR DESCRIPTION
**Pull request by charliewang95**
### Id of the task, bug, story or other reference.
[Bug 3547532](https://dynamicscrm.visualstudio.com/OneCRM/_workitems/edit/3547532): [V2] HTML welcome system message - Chat widget v2 no longer supports html-tags

### Description
HTML tags are not applied to system messages

## Solution Proposed
Set the text as html innerText.

Because the sanitization middleware is executed before activityMiddleware, dangerouslySetHtml is ok to use here. It is the same implementation inside webchat

### Acceptance criteria
_Define what are the conditions to consider the PR has achieved the intended goal_
1. html tags render correctly
2. normal texts doesn't have regression
3. No security concerns - scripts are not executed

## Test cases and evidence
_Include what tests cases were considered, any evidence of testing for future references, to identify any corner cases, etc_
proof1
proof2
proof3

### Sanity Tests
-   [ ] You have tested all changes in Popout mode
-   [ ] You have tested all changes in cross browsers i.e Edge, Chrome, Firefox, Safari and mobile devices(iOS and Android)
-   [ ] Your changes are included in the [CHANGELOG](../CHANGE_LOG.md)

## A11y
- [ ] You have run the [Automated Accessibility Check](https://accessibilityinsights.io/docs/en/windows/getstarted/automatedchecks) and have no reported issues

__*Please provide justification if any of the validations has been skipped.*__

### CODE REVIEW

File: chat-widget/src/components/webchatcontainerstateful/webchatcontroller/middlewares/renderingmiddlewares/activityMiddleware.tsx [hunk 1]
```diff
@@ -46,11 +46,9 @@ const handleSystemMessage = (next: any, args: any[], card: any, systemMessageSty
         return () => false;
     }
 
-    // eslint-disable-next-line react/display-name, @typescript-eslint/no-explicit-any
+    // eslint-disable-next-line react/display-name
     return () => (
-        <div key={card.activity.id} style={systemMessageStyles} aria-hidden="true">
-            {card.activity.text}
-        </div>
+        <div key={card.activity.id} style={systemMessageStyles} aria-hidden="true" dangerouslySetInnerHTML={{ __html: card.activity.text }}/>
```
Comment (akhiloct1510) [inline chat-widget/src/components/webchatcontainerstateful/webchatcontroller/middlewares/renderingmiddlewares/activityMiddleware.tsx:51]: Can it raise any Security Risk ?
For e.g.,
dangerouslySetInnerHTML={{ __html: content }}
content = '';

Comment (charliewang95) [reply chat-widget/src/components/webchatcontainerstateful/webchatcontroller/middlewares/renderingmiddlewares/activityMiddleware.tsx:51]: Why is this a security risk?

Comment (charliewang95) [reply chat-widget/src/components/webchatcontainerstateful/webchatcontroller/middlewares/renderingmiddlewares/activityMiddleware.tsx:51]: image

Comment (charliewang95) [reply chat-widget/src/components/webchatcontainerstateful/webchatcontroller/middlewares/renderingmiddlewares/activityMiddleware.tsx:51]: I can repro the attack. But since the message can only be added by admin, is there need to check for script tags?

Comment (akhiloct1510) [reply chat-widget/src/components/webchatcontainerstateful/webchatcontroller/middlewares/renderingmiddlewares/activityMiddleware.tsx:51]: Couple of things 

1. Does Admin allow to add the malicious content? If yes, we should check with them if this should be a bug at their end or what is their expectation if they are allowing this.
2. From LCW side, I still think, we should take care of XSS attacks in all such type of fixes.

Let's check Naresh's point of view on this

Comment (charliewang95) [reply chat-widget/src/components/webchatcontainerstateful/webchatcontroller/middlewares/renderingmiddlewares/activityMiddleware.tsx:51]: Actually nvm. Used the existing `escapeHtml` function
image
image

**Review by xTEddie — APPROVED** [review]

**Review by akhiloct1510 — APPROVED** [review]
